gui/boosterView.py

```diff
@@ -27,7 +27,8 @@
 
 
 class BoosterViewDrop(wx.PyDropTarget):
-    def __init__(self, dropFn):
+    def __init__(self, dropFn, *args, **kwargs):
+        super(BoosterViewDrop, self).__init__(*args, **kwargs)
```

Comment:
> If we're not using what the parent sets, then often it's a really good indication that we shouldn't be using that as a parent.

This, in general, is not true. If we inherit something and the parent __requires__ it as part of it's `__init__`, then absolutely the child class should also either have it in it's own `__init__` definition or call the parent class with some known initial values. There's no other way around it, considering it's required. But if it's an optional parameter, then it's unnecessary and, in my view, clutters up the code. `*args, **kwargs` is by definition an optional parameter.

Take all of our `wxWindow` definitions. We inherit `wx.Panel`, `wx.Frame`, etc, but we don't define the same things the parent defines "just in case we need them in the future". All we do (usually) is define the parent window, and then call the parent with the required parameters (along with our implementation of optional parameters depending on what we need for this Window). ie:

```
class FighterView(wx.Panel):
    def __init__(self, parent):  # note lack of parameters
        wx.Panel.__init__(self, parent, id=wx.ID_ANY, pos=wx.DefaultPosition, style=wx.TAB_TRAVERSAL)
```

I would rather keep things explicit (as you keep advocating for) - if our class implementation doesn't require us to pass in arguments to the parent, then don't. Otherwise we can have situations where I or some other contributor is trying to figure out why we have our implementation defined with  `*args, **kwargs` but we aren't using them anywhere. And I definitely don't want us to get into the habit of littering our wx definitions with `*args, **kwargs`.

Please change this and the extra call to the parent, and I believe we should be good to go with this PR. :)